pages/Javascript-API.md

```diff
@@ -99,8 +90,8 @@ scene.screenshot().then(function(screenshot) { window.open(screenshot.url); });
 #### `setActiveCamera(camera)`
 Sets the active camera to the camera specified by name, as named in the scene file.
 
-#### `setDataSource(name, source)`
-Loads a new `[data source](sources.md)` or replaces an existing one, and rebuilds the scene.
+#### `setDataSource(_string_ name, _object_ config)`
```

Comment:
I think I actually took these data types out (`_string_`, etc.) awhile ago, because while they are useful to have somewhere, they were not being formatted correctly in the doc generation (where they show up in the left-hand nav), and are also not consistent throughout these pages. So we should find a standard that works.